xbmc/cores/VideoPlayer/DVDCodecs/Overlay/OverlayCodecWebVTT.cpp

```diff
@@ -66,7 +66,7 @@ OverlayMessage COverlayCodecWebVTT::Decode(DemuxPacket* pPacket)
 
   m_webvttHandler.Reset();
 
-  m_webvttHandler.SetPeriodStart(pPacket->m_ptsOffsetCorrection);
+  m_webvttHandler.SetPeriodStart(pPacket->m_ptsOffsetCorrection * -1);
```

Comment:
Hum...shouldn't be the input stream add-on job to set the proper offset values instead of the core reversing it just because so? What if you have a negative offset?

Has this anything to do with the offset set on the player settings when you adjust subtitles?